### ISSUE TITLE
lineargradient ignores all but first and last defined colorstop

### ISSUE THREAD

**Issue by krayj** [AmoebeLabs/flex-horseshoe-card#40]
observed on release version: v 0.9.0

[repro]
1. set horseshoe fill style to use: lineargradient
2. provide a list of numerous (several more than 2) colorstops

[result]
1. the gradient is created using only the first and last defined colorstop, none of the in-between colorstop colors are used or displayed

[expect]
1. multiple defined colorstops should result in multiple gradients between each of the defined colorstop values

**Comment by skarchmit** [comment]
I agree with this one, but the documentation currently states that only the first and last are used. I wish it were the case as you described.

**Comment by krayj** [comment]
LOL. When I originally wrote that (over a year ago) the documentation and even the example code showed more than 2 color gradients in use...and working, and more than 2 were working in earlier versions. I wrote that bug due to an observed regression in behavior.

In the time since, the documentation was modified to match the new flawed behavior. Weird way to fix a bug, but OK.
